types.py

```diff
@@ -5,18 +5,29 @@
 # the Apache 2.0 License: http://www.apache.org/licenses/LICENSE-2.0
 
 
+import builtins
+import sys
+import typing
+import typing_extensions
+
+
 __all__ = (
     'BitString', 'Point', 'Path', 'Polygon',
     'Box', 'Line', 'LineSegment', 'Circle',
 )
 
+_BS = typing.TypeVar('_BS', bound='BitString')
```

Comment:
I'd use full names here to make the code read better

```suggestion
_BitString = typing.TypeVar('_BS', bound='BitString')
```

Reply:
Sounds good. I'll make the update.